xml/bp_troubleshooting_report.xml

```diff
@@ -27,14 +27,41 @@
   </para>
 <screen>&prompt.user;rpm -q supportutils-plugin-ses</screen>
   <para>
-   If it is missing on the &ceph; server, install it with
+   If it is missing on the &ceph; server, install it with:
   </para>
 <screen>&prompt.root;zypper ref &amp;&amp; zypper in supportutils-plugin-ses</screen>
+  <para>
+    <literal>supportutils-plugin-ses</literal> will try to capture as much
+    useful information as it can about both the &ceph; cluster as a whole,
+    and about the host on which it is running. Some information, including
+    <command>ceph daemon status</command>, podman image information, rpm verification and some
+    relevant configuration data will be saved to the <filename>plugin-ses.txt</filename> file
+    inside the <command>supportconfig</command> tar archive. A lot more information, including
+    overall cluster status, logs, and diagnostic data for each currently
+    running &ceph; daemon will be saved to files in the &ceph; subdirectory
+    inside the supportconfig tar archive.
+  </para>
  </tip>
+ <para>
+   When <command>supportconfig</command> is run on an admin node, it will
```

Comment:
I could be wrong but I thought we had an entity for "admin node"...?

Reply:
We sure do: https://github.com/SUSE/doc-ses/blob/master/xml/entity-decl.ent#L64 thanks for that